app/components/collections/edit/license_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<fieldset>
+  <div class="form-check d-flex pt-2">
+    <div class="ps-2">
+      <%= form.radio_button :license_option, 'required', class: 'form-check-input' %>
```

Comment:
The required attribute is set with `required: true` for `form.radio_button`.

However, this will be client side validation which we don't want. Instead, can you set a default value in the form (which will set a default radio button)?

Reply:
I may be misunderstanding, but in this case, the string `'required'` is the value and relates to the label `license_option_required`...

Reply:
No, I misunderstood. This is correct.